pkg/minishift/docker/image/image_handler.go

```diff
@@ -16,13 +16,36 @@ limitations under the License.
 
 package image
 
+// ProgressStatus defines a status code for import/export operations.
+type ProgressStatus int
+
+const (
+	Ok ProgressStatus = iota
```

Comment:
A little bit of nitpicking perhaps; but Ok is a term that is about 99% of the cases written as `OK`. Since this is also a constant, it suits better. This makes it more obvious that is relates to

  * https://github.com/minishift/minishift/pull/1839/files#diff-0d2b6075538057707499cc3316570998R108
  * https://github.com/minishift/minishift/pull/1839/files#diff-0d2b6075538057707499cc3316570998R402

and the other situations as:
  * https://github.com/minishift/minishift/pull/1839/files#diff-0d2b6075538057707499cc3316570998R115
  * etc ...

Reply:
> but Ok is a term that is about 99% of the cases written as OK. Since this is also a constant,

So what's about the other constants? FAIL and CACHE_MISS? Or you want to just rename Ok? I thought we at some stage said we don't use the all capital spelling for constants. Either way, one would have an inconsistency. 

If you feel strongly about it, I rename it. I don't understand what you mean with the references code diffs though. I am not quite sure how Ok vs OK matters there.

Reply:
Rename ALL. In the rest of the code, these variables do not stand out or are clear in their origin.

Reply:
I renamed the constants. Not sure I follow your line of thoughts though ;-)